### PR TITLE
Added Home Page

### PR DESCRIPTION
**Pull request by nareshpoonia**
Feat -1 Home Page

### CODE REVIEW

File: styles/index.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@import url(/styles/utility-new.css);
+.hero-image {
+  width: 100vw;
+  height: 800px;
```
Comment (Shivanshsharma01) [inline styles/index.css]: can use rem here

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean and well structured. Suggested change can be implemented

**Review by RakshithBhat03 — APPROVED** [review]
Everything looks good. Please implement the change suggested by Shivansh

File: styles/index.css (hunk 1 above)
Comment (nareshpoonia) [reply styles/index.css]: Thanks for pointing that out, changes have been made.

**Pull request merged**